params/variants.toml

```diff
@@ -5926,22 +5926,20 @@ selector.cent = "barInterrupted"
 
 
 
-[prime.partial-derivative]
-sampler = "∂ ð"
-samplerExplain = "Partial derivative symbol and Lowercase Eth"
+[prime.lower-eth]
```

Comment:
We should be able to create _two_ prime variants: one for eth and insular-d, one for partial derivative sign. The Eth variant will be assigned to tag `VXAA`. This could avoid creating another major version.